openedx_tagging/core/tagging/models/base.py

```diff
@@ -431,6 +447,7 @@ class ObjectTag(models.Model):
     id = models.BigAutoField(primary_key=True)
     object_id = case_insensitive_char_field(
```

Comment:
I just noticed this, but why is `object_id` case insensitive? Shouldn't it be case sensitive? `abc` and `ABC` are usually different IDs.

Reply:
Ah ok.. I think that came from a misunderstanding about content UsageKeys.. I thought they were case insensitive.

@ChrisChV  Do you mind to fix this here? Sorry there's been so many little mistakes like this..

Reply:
@bradenmacdonald Are you ok with the other case-insensitive char fields defined here?

Reply:
Please fix that in a small, focused, separate PR. Just add it to the list you're making. I don't want to muddy the waters or delay this PR any longer than I already have, and smaller focused PRs are usually better.

I am not sure about `_name` but I guess it makes sense to be case-insensitive.
If `_value` holds IDs it should be case-sensitive I think.

Reply:
cf https://github.com/openedx/modular-learning/issues/84